packages/loader/container-loader/src/loader.ts

```diff
@@ -278,13 +292,24 @@ export type IDetachedBlobStorage = Pick<IDocumentStorageService, "createBlob" |
 export class Loader implements IHostLoader {
     private readonly containers = new Map<string, Promise<Container>>();
     public readonly services: ILoaderServices;
-    private readonly logger: ITelemetryLogger;
+    private readonly mc: MonitoringContext;
 
     constructor(loaderProps: ILoaderProps) {
         const scope = { ...loaderProps.scope as FluidObject<ILoader> };
         if (loaderProps.options?.provideScopeLoader !== false) {
             scope.ILoader = this;
         }
+
+        const subMc = mixinMonitoringContext(
+            DebugLogger.mixinDebugLogger("fluid:telemetry", loaderProps.logger, { all: { loaderId: uuid() } }),
+            ConfigProvider.create(
+                "Fluid",
+                [sessionStorageConfigProvider("Fluid").value, // Global override for all configs
```

Comment:
Should this namespace prefix be different? I think as is, the things we'd have to put inside `sessionStorageConfigProvider` would have to be 'Fluid.Fluid'

Reply:
so i've been looking at this and i think all the current namespace stuff is broken. the problem is similar to the above comment
> @andre4i you move session storage here, but i think it needs to be in `ConfigProvider.create` the driver, loader, and runtime all move at different deployment speeds, so it's likely that the loader will be behind so just setting it here will lead to other layers not having it if this change isn't there yet

basically, it's likely the loader won't provide config initially, so nested config won't get any prefix. this will lead to fully qualified config name changing over time, but i think we want stable names. I'm leaning toward config taking a fully qualified namespace, and just leaving it at that